app/javascript/components/artist_works.js

```diff
@@ -18,6 +21,19 @@ class ArtistWorks extends React.Component {
     }, (response) => {console.err(response)});
   }
 
+  handleChange(event) {
+    this.setState({ comment: event.target.value });
+  }
+
+  handleSubmit(event) {
+    const artist_id = this.props.artist.id;
+    const commissions_route = APIRoutes.commissions.create;
+    const buyer_id = this.props.buyer.id;
+
+    Requester.post(commissions_route, { "buyer_id": buyer_id, "artist_id": artist_id, "comment": this.state.comment })
```

Comment:
pull the json object out into a variable `const payload = { "buyer_id": buyer_id"...` with each line containing only 1 key-value pair